pom.xml

```diff
@@ -30,6 +30,9 @@
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-java</artifactId>
       <version>4.18.1</version>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>
+      <version>4.18.1</version>
```

Comment:
<div>
<h3>Code Review Agent Run <a href = https://github.com/Amruta101998/selenium-automation/pull/2/files#2122405823>#64877</a> - 05/21/2024, 04:52 pm</h3>

<p>🔴 High importance</p>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Duplicate dependency entries for 'org.seleniumhq.selenium' with 'selenium-java' artifact are added. This can lead to confusion and potential build issues if different versions are specified in the future.
</div>
<div id="fix">
<b>Fix:</b> Remove the duplicate dependency entries to ensure clean and maintainable build configurations.
</div>
<details>
<summary>
Code suggestion
</summary>
<div>
<pre><code>
 @@ -30,9 +30,6 @@
      <groupId>org.seleniumhq.selenium</groupId>
      <artifactId>selenium-java</artifactId>
      <version>4.18.1</version>
 -    <groupId>org.seleniumhq.selenium</groupId>
 -    <artifactId>selenium-java</artifactId>
 -    <version>4.18.1</version>
    </dependency>
</code></pre>
</div>
</details>
<h4><a href="https://github.com/Amruta101998/selenium-automation/pull/2/files#2122405823"><-- Code Review Agent Run#64877</a></h4>
<i>AI Code Review powered by <img src="https://bito.ai/wp-content/uploads/2023/10/Logo-Bito-Black-cropped.svg" alt="Bito Logo" width="50" height="25"></i>
</div>
<br>

<br>

</div>